src/components/network-chart/NetworkChartRow.js

```diff
@@ -69,22 +83,21 @@ const PHASE_OPACITIES = PROPERTIES_IN_ORDER.reduce(
   {}
 );
 
-type NetworkChartRowProps = {
-  +index: number,
-  +marker: Marker,
-  // Pass the payload in as well, since our types can't express a Marker with
-  // a specific payload.
-  +networkPayload: NetworkPayload,
-  +timeRange: StartEndRange,
-  +width: CssPixels,
-  +threadIndex: ThreadIndex,
-};
-
-type State = {|
-  pageX: CssPixels,
-  pageY: CssPixels,
-  hovered: ?boolean,
-|};
+function NetworkPhase({ name, previousName, value, duration, ...style }) {
```

Comment:
I find the `...style` syntax to be hard to reason with here on how to use this function. I think it would be more easier to maintain with a real type, maybe something like:

I think a type would be nice...

```
type {
  name: string
  previousName: string,
  value: string,
  duration: number,
  style: {[string]: string}
}
```

Reply:
I read through the rest of this commit, and I do think a type is warranted here.

Reply:
done